Library/Homebrew/utils/analytics.rb

```diff
@@ -1,34 +1,52 @@
-
-def analytics_anonymous_prefix_and_os
+def analytics_label
   @analytics_anonymous_prefix_and_os ||= begin
-    "#{OS_VERSION}, #{HOMEBREW_PREFIX.to_s.gsub(ENV["HOME"], "~")}"
+    os = OS_VERSION
+    prefix = ", non-/usr/local" if HOMEBREW_PREFIX.to_s != "/usr/local"
```

Comment:
Not sure if we want to make that still output `/usr/local` for the reverse case. (It would at least make the code more consistent with the screenshot that is also part of this PR.)